x/twap/keeper_test.go

```diff
@@ -14,11 +14,13 @@ import (
 
 // TODO: Consider switching this everywhere
 var (
-	denom0            = "token/B"
-	denom1            = "token/A"
-	defaultUniV2Coins = sdk.NewCoins(sdk.NewInt64Coin(denom0, 1_000_000_000), sdk.NewInt64Coin(denom1, 1_000_000_000))
-	baseTime          = time.Unix(1257894000, 0).UTC()
-	tPlusOne          = baseTime.Add(time.Second)
+	denom0                 = "token/B"
+	denom1                 = "token/A"
+	denom2                 = "token/C"
+	defaultUniV2Coins      = sdk.NewCoins(sdk.NewInt64Coin(denom0, 1_000_000_000), sdk.NewInt64Coin(denom1, 1_000_000_000))
```

Comment:
nit: Consider renaming this to `defaultTwoAssetCoins` for a clearer distinction and relationship to  `defaultThreeAssetCoins`